content/docs-v1/citation-bibliographies.md

```diff
@@ -33,72 +33,110 @@ entries:
+```yaml
+  bibliography:
+    displayShort: true
+```
 
+### Page-level Bibliography
 
+If you are using in-text citations your publication will automatically include a page-level bibliography listing all works that were cited on that page using the `{% cite %}` shortcode.
 
-## Display a Bibliography
+If a publication appears to have missing bibliographic entries there is most likely an incorrect or missing `{% cite %}` shortcode in the essay end notes. The `id` used in the `{% cite %}` shortcode must exactly match the `id` in the `references.yaml` files.
+
+If you want to suppress the page level bibliography but keep the in-text citations, go into the `content/_data/config.yaml` and change the value to `false`:
+
+```yaml
+  bibliography:
+    displayOnPage: false
+```
 
-Your publication will automatically include a page-level bibliography listing all works that were cited on that page using the `q-cite` shortcode. A heading can be customized to go above the page bibliography with the `biblioHeading` parameter in your `config.yaml` file. The default is "Bibliography".
+If you do not want to use in-text citations but still want to include a page-level bibiliography you have a two options:
 
-Additionally, to create a complete bibliography for your entire publication, from all the entries in the project's `references.yaml` file, you can create a `bibliography.md` file and use the `q-bibliography` shortcode. The resulting bibliography will be output as a page in your publication. By default, your bibliography will automatically be ordered according to the `full` value in your `references.yaml` file.
+- Add a page-level biblio using Markdown
+- List the relevant IDs from the `references.yaml` file in the page YAML. Keep in mind, this may be unwieldy if there are a lot of references. You would then use the `{% bibliography %}` shortcode to add the page-level bibliography at the end of the essay.
 
-```go
-{{</* q-bibliography */>}}
+```yaml
+title: Chapter Title
+weight: 10
+layout: page
+references:
+  - "West 1939"
+  - "Evans 1938"
+  - "Lynd 1929"
 ```
 
-{{< q-figure id="modern-bibliography" >}}
+For the latter option, the order they’re listed in doesn’t matter. Quire will sort them all alphabetically unless a `sort_as` attribute has been added in the `references.yaml` file.
 
+### Complete Bibliography
 
-### Display the Short Reference in Bibliographies
+Additionally, to create a complete bibliography for your entire publication, from all the entries in the project's `references.yaml` file, you would add a `bibliography.md` file to your `content` directory. Then insert the `{% bibliography %}` shortcode to this `.md` file. The resulting bibliography will be output as a page in your publication. By default, your bibliography will automatically be ordered according to the `full` value in your `references.yaml` file unless the `sort_as` attribute has been applied.
```

Comment:
See note above about using `layout: bibliography` instead

Reply:
It wouldn't. `lauyout: bibliography` will create a page that has the full bibliography for the whole book (all the references in references.yaml). I don't think there's a use case for having that full biblio appear on an essay or entry page. Those will have their own page-level biblios based on usage of `% cite %`.